cmd/gossamer/commands/root.go

```diff
@@ -260,78 +271,54 @@ Expected format is 'URL VERBOSITY', e.g. ''--telemetry-url wss://foo/bar:0, wss:
 	return nil
 }
 
-// addLogFlags adds the log flags to the command
-func addLogFlags(cmd *cobra.Command) error {
-	if err := addStringFlagBindViper(cmd,
-		"lcore",
-		config.Log.Core,
-		"Core module log level",
-		"log.core"); err != nil {
-		return fmt.Errorf("failed to add --lcore flag: %s", err)
-	}
-
-	if err := addStringFlagBindViper(cmd,
-		"ldigest",
-		config.Log.Digest,
-		"Digest module log level",
-		"log.digest"); err != nil {
-		return fmt.Errorf("failed to add --ldigest flag: %s", err)
-	}
-
-	if err := addStringFlagBindViper(cmd,
-		"lsync",
-		config.Log.Sync,
-		"Sync module log level",
-		"log.sync"); err != nil {
-		return fmt.Errorf("failed to add --lsync flag: %s", err)
-	}
+func parseLogLevel() error {
+	// set default log level
+	moduleToLogLevel := map[string]string{
+		"global":  cfg.DefaultLogLevel,
+		"core":    cfg.DefaultLogLevel,
+		"digest":  cfg.DefaultLogLevel,
+		"sync":    cfg.DefaultLogLevel,
+		"network": cfg.DefaultLogLevel,
+		"rpc":     cfg.DefaultLogLevel,
+		"state":   cfg.DefaultLogLevel,
+		"runtime": cfg.DefaultLogLevel,
+		"babe":    cfg.DefaultLogLevel,
+		"grandpa": cfg.DefaultLogLevel,
+		"wasmer":  cfg.DefaultLogLevel,
+	}
+
+	customLog := viper.GetString("logs")
+
+	if len(customLog) > 0 {
+		logConfigurations := strings.Split(customLog, ",")
+
+		for _, config := range logConfigurations {
+			parts := strings.SplitN(config, ":", 2)
```

Comment:
Instead of the format `module:loglvl`, let's use `module=loglvl` since that's how substrate does it

Reply:
you mentioned the `module:loglvl` format in the description of this issue. 
pushed the changes for `module=loglvl` format.

Reply:
Thank you for making the change @axaysagathiya !